platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -729,6 +729,14 @@ The `model` and `component` tasks report on the structure of legacy software mod
 Previously, these tasks were automatically added to the project for every build.
 These tasks are now only added to a project when a rule-based plugin is applied (such as those provided by Gradle's support for building native software).
 
+=== Potential breaking changes
+
+==== Upgrade to Eclipse JGit 7.2.1
```

Comment:
Can you move this to the `=== Toolchain, Language, and Compatibility Changes` section instead?

Reply:
I moved it to that section, noticing that you renamed it in
* https://github.com/gradle/gradle/pull/33745

PTAL again in time for RC1